src/_content/organizers/emmanuel-asuah-danful.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Emmanuel Asuah Danful
+photo: emmanuel-asuah-danful.jpg
```

Comment:
@Ti88m as a heads up, the filename doesn't match this. It looks like there may be an issue with the extension being duplicated.

Reply:
@tim-schilling Thanks for the heads up.